Logger/BaseLoggerMixins.cs

```diff
@@ -1,7 +1,89 @@
-﻿namespace Logger
+﻿using System.Security.Cryptography;
+using System.Text;
+
+namespace Logger
 {
     public static class BaseLoggerMixins
     {
-        
+        public static void Error(this BaseLogger baseLogger, string message, params object[] objects)
+        {
+            if(baseLogger == null)
+            {
+                throw new System.ArgumentNullException(nameof(baseLogger));
+            }
+            if(objects != null)
+            {
+                StringBuilder sb = new StringBuilder(message);
+                for (int i = 0; i < objects.Length; i++)
+                {
+                    sb.Replace("{" + i + "}", objects[i].ToString());
+                }
+                baseLogger.Log(LogLevel.Error, sb.ToString());
+            }
+            else
+            {
+                baseLogger.Log(LogLevel.Error, message);
```

Comment:
```suggestion
                baseLogger.Log(LogLevel.Error, string.Format(message, args));
```
Not quite sure if that actually works, but a call to string.Format() should do the trick. 
Note, args is the existing parameter "objects". Just using a more clear name.